src/main/java/com/techbirdssolutions/pos/PosApplication.java

```diff
@@ -3,11 +3,19 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.sql.SQLOutput;
+
 @SpringBootApplication
-public class PosApplication {
+public class PosApplication
```

Comment:
follow the standard ad class name an { in same line

Reply:
resolved